src/background/refreshMenus.ts

```diff
@@ -31,14 +31,42 @@ export async function refreshMenus() {
       contexts: ["selection"],
       title: `Send selection to ${chatName} ${chat.type}`,
       onclick: async (info, tab) => {
-        const data = await sendTabMessage(tab.id!, "getSelection", undefined)
```

Comment:
This was returning all the images found in a selection or undefined if none found. It was useful for sending a twitter tweet with multiple images.

I would leave that in and only fallback to sending just text when there's no images in selection (data is null/undefined)

Reply:
Ahh, I wasn't able to get that to work but I hadn't tried selecting from a site like Twitter, I'll do more testing. In the situation where both text and images are selected what kind of behavior would you like? (Both text and images are sent or just one or the other)